### ISSUE TITLE
Components / Forms elements - Element IDs and Contrast

### ISSUE THREAD

**Issue by yesenia-gtz** [Office-of-Digital-Services/California-State-Web-Template-Website#974]
[Components / Forms elements
](https://as-cdt-pub-migr-betaws-ww-p-001-stage.azurewebsites.net/components/form-elements.html)

**1. Color contrast is not sufficient**
- [ ] Text on this page does not meet WCAG minimum color-contrast requirements.Change Text field text contrast to: #004261

![Screen Shot 2022-12-13 at 1 18 30 PM](https://user-images.githubusercontent.com/98193284/207445602-539c4385-280e-40a9-8aa3-2a579d7f3bd5.png)

**2. Element IDs (#tab-content) are identical and can confuse when running with a screen reader.** 

**#tab-content**
#InputName
**#tab-content**
#InputName
**#tab-content
#tab-content
#tab-content
#tab-content
#tab-content**

![Screen Shot 2022-12-13 at 1 15 58 PM](https://user-images.githubusercontent.com/98193284/207445199-4730c94a-5b15-411f-90fd-5a59a8a2189c.png)

**RECOMMENDATION:**
- [ ] Make element IDs unique.

#tab-content-1
#InputName
#tab-content-2
#InputName
#tab-content-3
#tab-content-4
#tab-content-5
#tab-content-6
#tab-content-7
